pod/perldelta.pod

```diff
@@ -27,6 +27,25 @@ here, but most should go in the L</Performance Enhancements> section.
 
 [ List each enhancement as a =head2 entry ]
 
+=head2 Subroutine signatures are no longer experimental
+
+Introduced in Perl version 5.20.0, and modified several times since, the
+subroutine signatures feature (C<use feature 'signatures'>) is now no longer
+considered experimental. It is now considered a stable language feature and
+is included in the C<:5.36> feature bundle, enabled automatically by
+C<use v5.36>:
+
+    use v5.36;
+
+    sub add ($x, $y) {
+      return $x + $y;
+    }
+
+No longer prints a warning.
```

Comment:
slightly awkward sentence fragment - perhaps this could just be included in the previous paragraph: "is now no longer considered experimental and no longer prints a warning."

Reply:
Reworded and force-pushed.